### PR TITLE
Switch to allowed actions list rather than ignored actions

### PR DESCRIPTION
**Pull request by lennym**
This makes it safer when dealing with future changes.

### CODE REVIEW

File: lib/recipients/project.js [hunk 1]
```diff
@@ -9,12 +9,16 @@ module.exports = async ({ schema, logger, task }) => {
   const applicantId = get(task, 'data.subject');
   const establishmentId = get(task, 'data.establishmentId');
   const action = get(task, 'data.action');
-  const ignoredActions = ['create', 'delete', 'delete-amendments', 'fork', 'convert'];
+  const allowedActions = [
+    'grant',
+    'update',
+    'revoke'
+  ];
```
Comment (wheelsandcogs) [inline lib/recipients/project.js:16]: no notifications for ppl transfer?

Comment (lennym) [reply lib/recipients/project.js:17]: _handwaves_

File: lib/recipients/pil.js [hunk 2]
```diff
@@ -9,7 +9,13 @@ module.exports = ({ schema, logger, task }) => {
   const applicantId = get(task, 'data.subject');
   const establishmentId = get(task, 'data.establishmentId');
   const action = get(task, 'data.action');
-  const ignoredActions = ['create', 'delete'];
+  const allowedActions = [
+    'transfer',
+    'grant',
+    'update',
+    'revoke',
+    'review'
```
Comment (joefitter) [inline lib/recipients/pil.js:17]: update-conditions?

Comment (joefitter) [reply lib/recipients/pil.js:17]: can't suggest as would need a comma before

**Pull request merged**